multiraft/multiraft.go

```diff
@@ -159,10 +159,10 @@ func NewMultiRaft(nodeID roachpb.NodeID, storeID roachpb.StoreID, config *Config
 		Events: make(chan []interface{}),
 
 		// Input channels.
-		reqChan:         make(chan *RaftMessageRequest, reqBufferSize),
-		createGroupChan: make(chan *createGroupOp),
-		removeGroupChan: make(chan *removeGroupOp),
-		proposalChan:    make(chan *proposal),
+		reqChan:         make(chan RaftMessageRequest, reqBufferSize),
```

Comment:
Buffered channels statically allocate their buffer, so changing a buffered channel from pointers to values has a larger memory cost. Also, using a shallow copy of protobufs can be misleading, so I would keep this one a pointer.